rust/processor/src/db/common/models/ans_models/parquet_ans_lookup_v2.rs

```diff
@@ -0,0 +1,172 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::{
+        ans_models::{
+            ans_lookup::{AnsPrimaryName, CurrentAnsPrimaryName},
+            ans_utils::SetReverseLookupEvent,
+        },
+        token_v2_models::v2_token_utils::TokenStandard,
+    },
+};
+use allocative_derive::Allocative;
+use aptos_protos::transaction::v1::Event;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Default, Debug, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct AnsPrimaryNameV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub registered_address: String,
+    pub token_standard: String,
+    pub domain: Option<String>,
+    pub subdomain: Option<String>,
+    pub token_name: Option<String>,
+    pub is_deleted: bool,
+    #[allocative(skip)]
```

Comment:
@ying-w let me know if it's missing any fields

Reply:
missing `expiration_timestamp` and `subdomain_expiration_policy`

Reply:
@rtso do you know how to parse out the expiration timestamps?

Reply:
Primary names don't have expiration timestamp, only ans lookups do. This PR looks like it's missing moving ans_lookup_v2 to Parquet @yuunlimm

Reply:
https://github.com/aptos-labs/aptos-indexer-processors/blob/e35189535ecce84eb654bc1fb3ea02ab72b6f52d/rust/processor/src/processors/ans_processor.rs#L737

Reply:
I also realized that this processor is still writing to the v1 ANS tables 
```
all_current_ans_lookups,
all_ans_lookups,
all_current_ans_primary_names,
all_ans_primary_names,
```
We can include these in the deprecated tables and turn it off

Reply:
I don't think ans_lookup_v2 was a part of parquet migration.

Reply:
@ying-w do we need lookups in BQ?

Reply:
just realized this was for `ans_primary_name_v2` not `ans_lookup_v2`

Yes we use both `ans_lookup_v2` and `ans_primary_name_v2`